mesheryctl/internal/cli/root/system/model.go

```diff
@@ -413,6 +425,133 @@ mesheryctl exp model search [query-text]
 	},
 }
 
+var importModelCmd = &cobra.Command{
+	Use:   "import",
+	Short: "import model",
+	Long:  "import a model from the registry",
+	Example: `
+
+// Import a model
+mesheryctl exp model import --file <URI>
+	`,
+	Args: func(_ *cobra.Command, args []string) error {
+		const errMsg = "Usage: mesheryctl exp model import <URI/path to Model>\nRun 'mesheryctl exp model import --help' to see detailed help message"
```

Comment:
No angle brackets please. 
```suggestion
		const errMsg = "Usage: mesheryctl exp model import [URI/path to Model]\nRun 'mesheryctl exp model import --help' to see detailed help message"
```